src/instructlab/data/generate.py

```diff
@@ -0,0 +1,218 @@
+
+import click
+from instructlab import helpers
+import logging
+
+@click.command()
+@click.option(
+    "--model",
+    default=helpers.DEFAULT_MODEL,
+    show_default=True,
+    help="Name of the model used during generation.",
+)
+@click.option(
+    "--num-cpus",
+    type=click.INT,
+    help="Number of processes to use.",
+    default=helpers.DEFAULT_NUM_CPUS,
+    show_default=True,
+)
+@click.option(
+    "--chunk-word-count",
+    type=click.INT,
+    help="Number of words to chunk the document",
```

Comment:
```suggestion
    help="Number of words to chunk the document",
```
This could be better IMO

Reply:
This is not new content, though

Reply:
I wouldn't block on it, but if there's a better way to explain I don't see why not improve the text here - similar to https://github.com/instructlab/instructlab/pull/990/files#r1626438457

But again, not a must-have

Reply:
I agree, no new content. Making this a heavier lift is not what we need